### PR TITLE
RHDEVDOCS-5907: Updated the shp CLI command examples

### PR DESCRIPTION
**Pull request by shipsing**
**Purpose**: To resolve this issue: https://issues.redhat.com/browse/RHDEVDOCS-5907

**Aligned team**: DevTools

**Version for cherrypicking**: build-docs-1.0, build-docs-1.1

**Content for preview**:  
Work with Builds: https://71330--docspreview.netlify.app/openshift-builds/latest/work_with_builds/using-builds

**SME review**:  @apoorvajagtap @adambkaplan 

**QE review**:  @jitendar-singh 

**Peer review**:  @smunje1

### CODE REVIEW

File: modules/ob-release-notes-1-0.adoc
Comment (apoorvajagtap) [inline modules/ob-release-notes-1-0.adoc:45]: I am not sure if adding this line makes any change. I mean, we are releasing the CLI aka shp, which itself indicates that one can use `shp` or `oc` to manage the builds (Also "Work with Builds" section makes it evident). So, I think we can skip adding this line.
But, if we still feel that it's good to have this specified, feel free to proceed.

**Review by apoorvajagtap — APPROVED** [review]

File: modules/ob-release-notes-1-0.adoc
Comment (shipsing) [reply modules/ob-release-notes-1-0.adoc]: removed this point, Thanks

**Comment by smunje1** [comment]
@shipsing This changes LGTM. As I am new to the peer review squad, I request that you wait until @bscott-rh gives approval.

**Comment by bscott-rh** [comment]
> @shipsing This changes LGTM. As I am new to the peer review squad, I request that you wait until @bscott-rh gives approval.

LGTM

**Review by adambkaplan — APPROVED** [review]
LGTM!

File: modules/ob-creating-a-buildah-build.adoc
Comment (mjpytlak) [inline modules/ob-creating-a-buildah-build.adoc:96]:
```suggestion
<1> Optional: By using the `--follow` flag, you can view the build logs in the output result.
```
I am understanding correctly, this flag is optional. If so, this is the correct standard to follow. Please also apply this update to `modules/ob-creating-a-s2i-build.adoc`.

**Review by mjpytlak — COMMENTED** [review]
@shipsing One item to address ahead of a merge. Thank you.

File: modules/ob-creating-a-buildah-build.adoc
Comment (shipsing) [reply modules/ob-creating-a-buildah-build.adoc:96]: Hi @mjpytlak, Thank you for this suggestion. Will it be okay if i cover this suggestion in a new PR for which i have to work. I have recently migrated to Fedora and need some time to set up Git-realted stuff.
This PR is a little bit urgent. Could you please merge it for now.

Comment (mjpytlak) [reply modules/ob-creating-a-buildah-build.adoc:96]: @shipsing Understood. Yes, I will merge this for you. Please address the final comment in a separate PR.

Comment (shipsing) [reply modules/ob-creating-a-buildah-build.adoc:96]: Resolving this issue as part of this PR: https://github.com/openshift/openshift-docs/pull/71602

**Pull request merged**
